README.md

```diff
@@ -0,0 +1,34 @@
+# apostrophe-link-widgets
+
+A link widget for [ApostropheCMS](http://apostrophecms.org/).
+
+`apostrophe-links-widgets` provides schema and a view for links.
+
+## Installation
+
+```bash
+npm install apostrophe-link-widgets --save
+```
+
+## Use
+
+You can use `apostrophe-links-widgets` as a singleton, area or as type `singleton` in your schema.
+
+This widget is useful for creating links, buttons or navigation lists.
+
+```nunjucks
+{{ apos.singleton(data.page, 'link' 'apostrophe-link') }}
```

Comment:
Missing comma, please test the example code. People use it and they tend to walk away if it doesn't run.